src/dash-panels/_allPanels.ts

```diff
@@ -19,6 +19,7 @@ import text from './text.vue'
 import transit from './transit.vue'
 import vega from './vega.vue'
 import video from './video.vue'
+import overview from './overview.vue'
```

Comment:
Please put these in alphabetical order, I think the new ones are incorrectly at the bottom